arelle/Cntlr.py

```diff
@@ -179,12 +179,10 @@ def __init__(self, hasGui=False, logFileName=None, logFileMode=None, logFileEnco
                 self.userAppDir = os.path.join( impliedAppDir, "Arelle")
             if hasGui:
                 try:
-                    import win32clipboard
```

Comment:
Arbitrary line.

I appreciate the PR. I'd also like to see these cleaned up, but there's considerable risk with this PR. Imports have side effects and some are used for feature detection (such as this one).  Someone would need to regression test many parts of Arelle (including plugins and across multiple platforms). It would also cause many merge conflicts for long lived branches (such as the table linkbase certification work). I don't think the benefits of doing this in one PR outweigh the amount of work necessary to get it merged.

Incrementally cleaning up imports as modules are touched (and need to be tested anyway) is a better path forward.

Reply:
I fully understand your concerns and appreciate your feedback. This PR was created to be in line with the suggestion of not having the clutter of white space cleanup.

I'll close this and cleanup as I go.